addons/website_sale/data/mail_template_data.xml

```diff
@@ -39,7 +39,7 @@
                         <hr/>
                     </t>
                     <div style="text-align: center; padding: 16px 0px 16px 0px; font-size: 14px;">
-                        <a t-attf-href="{{ object.get_base_url() }}/shop/cart?access_token={{ object.access_token }}"
+                        <a t-attf-href="{{ object.get_base_url() }}/shop/cart?id={{ object.id }}&amp;access_token={{ object.access_token }}"
```

Comment:
upgrade -> if_unchanged update record

Reply:
Indeed, I've checked and `util.if_unchanged(cr, "website_sale.mail_template_sale_cart_recovery", util.update_record_from_xml)` is already present in the upgrade file.